suites/pacific/rados/tier-3_rados_mondb_trimming.yaml

```diff
@@ -0,0 +1,173 @@
+# Suite contains tests related to mon db trimmings
+# Note: the test cannot be run on a cluster that is just created as the DB size increases with
+# new mappings for around next hour or so.. This test needs to be run on cluster who's age is at least 1 hour.
+
+tests:
+  - test:
+      name: setup install pre-requisistes
+      desc: Setup phase to deploy the required pre-requisites for running the tests.
+      module: install_prereq.py
+      abort-on-fail: true
+
+  - test:
+      name: cluster deployment
+      desc: Execute the cluster deployment workflow.
+      module: test_cephadm.py
+      config:
+        verify_cluster_health: true
+        steps:
+          - config:
+              command: bootstrap
+              service: cephadm
+              base_cmd_args:
+                verbose: true
+              args:
+                custom_repo: "cdn"
+                registry-url: registry.redhat.io
```

Comment:
Let's avoid having the registry url hardcoded... in IBM-C we use a different image registry.

Reply:
removed the registry link